README.md

```diff
@@ -417,29 +417,14 @@ assert cur.description[0][1] == "timestamp with time zone"
 
 Start by forking the repository and then modify the code in your fork.
 
-Clone the repository and go inside the code directory. Then you can get the
-version with `./setup.py --version`.
+Clone the repository and go inside the code directory.
 
-We recommend that you use Python3's `venv` for development:
+Python dependencies are managed using [Poetry](https://python-poetry.org/) which helps to ensure the project is managed in a deterministic way. Poetry [creates a virtual environment](https://python-poetry.org/docs/managing-environments/) to aid with the process. Currently this project uses [dependency groups](https://python-poetry.org/docs/master/managing-dependencies/) which are a pre-release feature (denoted by the `--preview` flag) and thus Poetry should be installed via:
 
 ```
-$ python3 -m venv .venv
-$ . .venv/bin/activate
-$ pip install .
+$ curl -sSL https://install.python-poetry.org | python3 - --preview
```

Comment:
Let's wait for the official release before merging.

Reply:
See previous comment.